autoload/neomake.vim

```diff
@@ -368,9 +371,9 @@ function! s:AddExprCallback(maker) abort
 
     if list_modified
         if file_mode
-            call setloclist(a:maker.winnr, list, 'r')
+            call setloclist(a:maker.winnr, list, 'r', name)
```

Comment:
@nerdrew Hey, I'm the one who added `{title}` to these functions.

> I'm not sure how to test it.

Testing what exactly?